samples/client/main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"time"
 )
 
+//dd:span
```

Comment:
If I am not mistaken the samples tests do not test if traces are outputted. Wouldn't it be better to put this in the dd:span integration test ?

Reply:
We cannot have a `func main()` in tests... we could *maybe* do something by doing an `orchestrion go run` in there...

Reply:
No no no. This is already an weird edge case, your explanation is enough